server/engine/src/main/java/org/eclipse/lsp/cobol/service/delegates/communications/ServerCommunications.java

```diff
@@ -152,7 +152,7 @@ private void notifyWorkProgressBegin(String uri) {
     WorkDoneProgressBegin workDoneProgressBegin = new WorkDoneProgressBegin();
     workDoneProgressBegin.setTitle(messageService.getMessage(
             "Communications.syntaxAnalysisInProgressTitle",
-            files.getNameFromURI(uri)));
+            files.getNameFromURI(files.decodeURI(uri))));
```

Comment:
does it work reasonably well with `untitled:` or e4e schemas?

Reply:
schemas are ignored

Reply:
query string?

Reply:
Both getNameFromURI implementations I found seem to treat the input as a path, not URI.

Reply:
Do you think I should reimplement them too?

Reply:
That we will address in separate PR.